spec/requests/sign_in_spec.rb

```diff
@@ -27,10 +27,7 @@
   end
 
   context "when the users is not known by the application" do
-    before do
-      User.create!(email: "another.user@education.gov.uk")
```

Comment:
This is a change in the behaviour of the test; what was the significance (or given the fact that deleting it still passes, what should the significance have been) of `another.user@education.gov.uk`?

Reply:
Ah, yeah, I forgot about that. That line doesn't seem to do anything. 

I'll double check what the test is trying to assert and rewrite if necessary 👀